.jenkins

```diff
@@ -0,0 +1,52 @@
+pipeline {
+    agent {
+        label 'DoNotRunAnythingOnMaster'
+    }
+    stages {
+        stage('Build') {
+            parallel {
+                stage('CUDA-10.1-NVCC') {
```

Comment:
Why call it `NVCC` if it's already `CUDA`?

Reply:
cause I planned on having clang + cuda